Dockerfile

```diff
@@ -12,13 +12,16 @@ RUN go mod download
 # Copy the go source
 COPY . ./
 
+RUN go mod tidy
```

Comment:
Because mod tidy updates dependencies we might get to the point where automatic image builds results in dependencies from container being different than dependencies in the repository.